tests/playTests.sh

```diff
@@ -310,6 +310,17 @@ test -f precompressedFilterTestDir/input.6.zst.zst
 println "Test completed"
 
 
+
+println "\n===>  warning prompt for decompressing multiple files does not swallow characters"
```

Comment:
nit: Is this comment accurate? This looks like it is triggering the overwrite prompt.

Reply:
Oops yeah - the overwrite prompt uses the same warning function as multifile decompression, I'll correct that.